### PR TITLE
[Chat][RTE] Table context menu update

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

- updated RoosterJs dependencies
- add icons props for the table context menu items
- removed cc flags form strybook page

# Why

# How Tested

Storybook

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by jpeng-ms** [comment]
@vhuseinova-msft  it seems like you can't edit second column if first column has bullet points, the focus will jump back to first column as soon as you type anything 

https://github.com/Azure/communication-ui-library/assets/109105353/ce558f76-4942-42b7-9452-35b887ca6ef2

**Comment by vhuseinova-msft** [comment]
> https://60c7ae6891f0e90039d7cd54-nndbrxjztz.chromatic.com/

@jpeng-ms hm... I can edit a cell in the second column in edge
could you share steps to reproduce? Could you also share what browser you use?

File: packages/react-components/src/components/styles/RichTextEditor.styles.ts [hunk 1]
```diff
@@ -308,3 +308,10 @@ export const insertTableMenuTitleStyles = mergeStyles({
   fontSize: '0.75rem',
   marginBottom: '0.5rem'
 });
+
+/**
+ * @private
+ */
+export const tableContextMenuIconStyles = mergeStyles({
+  marginTop: '6px'
```
Comment (palatter) [inline packages/react-components/src/components/styles/RichTextEditor.styles.ts:316]: I know we don't have clear answer yet for px/rem but for consistency, should we put this in rem?

**Review by palatter — APPROVED** [review]

File: packages/react-components/src/components/styles/RichTextEditor.styles.ts (hunk 1 above)
Comment (vhuseinova-msft) [reply packages/react-components/src/components/styles/RichTextEditor.styles.ts:316]: Left it px because of this :) sure, will update!

**Comment by vhuseinova-msft** [comment]
> @vhuseinova-msft it seems like you can't edit second column if first column has bullet points, the focus will jump back to first column as soon as you type anything
> 
>  bug.mov

John has helped to find steps to reproduce (the issue exists only for RTE edit box). The issue will be addressed in a separate PR

**Review by Leah-Xia-Microsoft — APPROVED** [review]

**Pull request merged**